data_conversion/extract_workflows_data.py

```diff
@@ -9,8 +9,9 @@
 WORKFLOW_FILE = 'workflows.csv'
 DATA_OUT_DIR = 'outputs'
 DATA_IN_DIR = 'inputs'
-EPS = 0.001
-MIN_SAMPLES = 3
+EPS_METERS = 50  # clustering size in meters
+EPS = EPS_METERS / 78710  # 1 deg ~= 78710 m
```

Comment:
I thought a degree was something like 111 km? But really this depends on coordinate and on the projection being used. Just noting to future-self that there's probably a bit of work to be done here finding the best SRS for a given lat/long and using that to convert to metres. I don't think it's critical right now as we have done a lot of near-equatorial work and haven't required that level of resolution. But in the future it might be.

Reply:
That is a good point, I just did a quick google search for a conversion and used the first number I saw.